packages/new-editor/src/components/editor/Editor.css

```diff
@@ -1,9 +1,9 @@
 .master-panel {
-  background-color: var(--N100);
+  background-color: var(--N75);
 }
 
 .master-toolbar {
-  background-color: var(--N100);
+  background-color: var(--N75);
```

Comment:
I'm not sure how the figma mockup looks like, but IMO this toolbar should look the same as in the other editors even if we don't have all the buttons on it... 
More like the example here (also with the collapse sidebar function): 
https://jenkins.ivyteam.io/job/ui-components/job/master/106/artifact/packages/components/storybook-static/index.html?path=/story/master-toolbar--with-title